api/src/db/model/request.ts

```diff
@@ -19,6 +19,7 @@ import { CommonFields, Model } from './model';
 export const enum RequestEditType {
     Contacts = 'contacts',
     Namespaces = 'namespaces',
+    Description = 'description',
```

Comment:
In the future when we have the edit capability for a profile's project name in place, are we planning on bundling the changes together for both project name and description upon as one request to the bot? If yes, perhaps the Description as the requestEditType might be a bit misleading; if no, please ignore this note. Thanks.